packages/paste-design-tokens/formatters/d.ts.ts

```diff
@@ -5,13 +5,24 @@ import {getTokenCategories} from '../utils/getTokenCategories';
 import {formatSingleTokensWithTemplate} from '../utils/formatSingleTokensWithTemplate';
 import {formatGroupTokensWithTemplate} from '../utils/formatGroupTokensWithTemplate';
 
+const isNumeric = (value: string): boolean => {
+  // To check if a string is a number, we check if it isn't a number
+  // eslint-disable-next-line no-restricted-globals
+  return !isNaN((value as unknown) as number);
+};
+
 export const tokenTemplate = ({name, value}: {name: string; value: string}): string =>
   `export declare const ${lodash.camelCase(name)} = "${value}";`;
 
 const categoryTemplate = (categoryName: string, props: Token[]): string => `export declare const ${lodash.camelCase(
   categoryName
 )}: {
-${props.map(prop => `  ${lodash.camelCase(prop.name)}: 'string';`).join('\n')}
+${props
+  .map(prop => {
+    const value = isNumeric(prop.value) ? prop.value : `"${prop.value}"`;
```

Comment:
Would be cool to add at least one token in the fixtures to test this logic and get it in the snapshot

Reply:
I pushed up z-index fixtures. Let me know if this is what you meant.

Reply:
Perfect 👍